PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+# Quality Assurance Checklist
+To make reviews more efficient, please make sure the board meets the following standards and check everything off once the board meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions.
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Minimum Prerequisites
+- [ ] The board passes ERC and DRC.
+    - Note some ERC warnings are acceptable (**_CAUTION_** please be careful)
+    - Note some DRC warnings are acceptable (less so than ERC: **_CAUTION_** please be careful).
+- [ ] All traces are routed.
+- [ ] Units are in metric.
+- [ ] Gerbers are in a zip file in the main directory (after physical design is done)
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Schematic Level Requirements 
+- [ ] Is proper noise resistance given to all peripheral devices (bypass caps and coils/ferrites)?  
+- [ ] Is proper ESD protection given to all MCU input pins (zener diodes)?  
+- [ ] Is proper power protection given to peripheral devices (zener diodes)?  
+- [ ] Are peripheral units used properly (reading datasheet)?  
+- [ ] Are testing points added at useful places?  
+- [ ] Is there proper short to GND protection at MCU outputs (inline resistors)?  
+- [ ] Do ADC inputs have caps?  
+- [ ] Are ADC inputs biased so there is room above expected value to determine if value is being overflowed?  
+- [ ] Are LED's located at useful places (comm, power, debugging, extra GPIO)?  
+- [ ] Are parts chosen easy to collect?  
+- [ ] Are parts chosen easy to solder?  
+- [ ] Is there reverse polarity protection on inputs?  
+- [ ] Do parts in the schematic match the actual part datasheet?
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## BOM Requirements
+- [ ] **Bill Of Materials**: is there an Excel Sheet BOM and a Mouser Cart link?
```

Comment:
Is Mouser our exclusive distributor? For the MPPT boards and likely others, there are some components only listed on Digikey. And for other engineers, potentially Robotshop, McMaster, etc.

Also do we have a format for the BOM? I'm typically generating it from KiCAD with the following columns:
![image](https://user-images.githubusercontent.com/31521956/236359367-1100e933-df84-4922-b0b7-cb4a879e339a.png)

Reply:
no, i just want links. we can have multiple.